nautobot_device_lifecycle_mgmt/forms.py

```diff
@@ -63,7 +63,9 @@ def prepare_value(self, value):
         return super().prepare_value(pk_list)
 
 
-class HardwareLCMForm(BootstrapMixin, CustomFieldModelForm, RelationshipModelForm):
+class HardwareLCMForm(
+    BootstrapMixin, CustomFieldModelForm, RelationshipModelForm
```

Comment:
In 1.4 these forms should probably inherit from `NautobotModelForm`, which combines `(BootstrapMixin, CustomFieldModelFormMixin, RelationshipModelFormMixin, NoteModelFormMixin)`

Reply:
What would be the best way to support these class definitions, so that 1.2.0 .. 1.4.0 are covered?

```
class CVELCMFilterSet(NautobotFilterSet, StatusModelFilterSetMixin, CustomFieldModelFilterSet):
class VulnerabilityLCMFilterSet(NautobotFilterSet, StatusModelFilterSetMixin, CustomFieldModelFilterSet):
```

Would something like that make sense?

```
try:
    from nautobot.extras.filters import NautobotFilterSet
except ImportError:
    # TODO: Remove once plugin no longer supports Nautobot < 1.4.0
    from nautobot.utilities.filters import BaseFilterSet
    from nautobot.extras.filters import CustomFieldModelFilterSet
    class NautobotFilterSet(BaseFilterSet, CustomFieldModelFilterSet):
        """Emulate NautobotFilterSet from Nautobot 1.4.0"""
```